src/utils.js

```diff
@@ -105,7 +105,7 @@ const parseScript = filename => {
   }
   const lines = content.split("\n");
   const filteredCommands = lines.filter(line => {
-    return line !== "" && !line.trim().startsWith("#");
```

Comment:
are you sure `line` and `line !== ""` are equivalent?

Reply:
In here `line` would always be a string value. Will switch back to `line !==""` as it's more clear